docs/level_0/basic_concepts.md

```diff
@@ -0,0 +1,79 @@
+# Basic Concepts
+
+This document describes the basic concepts, artifacts and their relationships of Landscaper. 
+
+## DeployItem
+
+The main objective of Landscaper is to set up large cloud environments with complex relationships consisting of many
+small installation tasks like helm deployments, network setup etc. These elementary installation tasks are defined by 
+DeployItems. There are different types of DeployItems for example for deploying helm charts, plain kubernetes manifests 
+or terraform configurations.
+
+A DeployItem can define import parameter for data needed for the installation task, e.g. the target cluster where 
+a helm chart should be deployed. A DeployItem can also define output parameter for data it creates and which can be 
+consumed by others.
+
+## Blueprint
+
+Several DeployItems can be collected in a Blueprint. In this sense, a Blueprint is a set of installation tasks. 
+
+A Blueprint can define an interface for import data, required by its DeployItems. It can also define an export 
+interface to expose output data of its DeployItems. Furthermore, the execution order of the DeployItems of a Blueprint 
+could be specified. 
+
+Blueprints are reusable collections of combined installation tasks which could be parameterized through their import 
+specification/parameters. They describe system setups consisting of a set of installations task and their dependencies. 
```

Comment:
this should be at the beginning of the blueprints section.
So that the reader gets a high level explenation of blueprints and then we can describe the blueprint in more detail with deployitems etc

Reply:
we have changed this